### PR TITLE
Removes CoinSwitch from Deposit screen, references to CoinSwitch

### PR DESCRIPTION
**Pull request by nchastain**
This pull request:
1. Removes all references to CoinSwitch, including messages, metrics events and its row on the "Deposit" view
2. When on mainnet, places the "Buy from Wyre" option above the "Direct Deposit Ether" option on the "Deposit" view (just a reordering, no functional changes). Because the button in from the "Main" view now reads "Buy" rather than "Deposit," it is more likely that users will arrive at this screen interested in purchasing ether, rather than depositing it.

### CODE REVIEW

**Comment by nchastain** [comment]
I have read the CLA Document and I hereby sign the CLA

File: ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js [hunk 1]
```diff
@@ -163,28 +137,28 @@ export default class DepositEtherModal extends Component {
             })}
             {this.renderRow({
               logo: (
-                <div
+                <img
+                  alt=""
                   className="deposit-ether-modal__logo"
+                  src="./images/deposit-eth.svg"
                   style={{
-                    backgroundImage: "url('./images/coinswitch_logo.png')",
```
Comment (Gudahtt) [inline ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js:169]: Could you remove this logo as well? This was the only place it was used

**Review by Gudahtt — COMMENTED** [review]
Great work! It looks OK at first glance. The only thing I noticed was the leftover logo.

File: ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js (hunk 1 above)
Comment (nchastain) [reply ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js:169]: @Gudahtt Good catch. Just added a new commit to delete the logo as well.

**Review by danjm — APPROVED** [review]
LGTM

**Pull request merged**